bin/tests/cli.rs

```diff
@@ -391,3 +398,194 @@ fn test_directory_symlinks_delete() {
     common::assert_agent_running(&mut agent_handle);
     agent_handle.kill().expect("Could not kill process");
 }
+
+#[test]
+#[cfg_attr(not(feature = "integration_tests"), ignore)]
+fn lookback_start_lines_are_delivered() {
+    let _ = env_logger::Builder::from_default_env().try_init();
+    let subject_alt_names = vec!["logdna.com".to_string(), "localhost".to_string()];
+
+    let cert = generate_simple_self_signed(subject_alt_names).unwrap();
+    // The certificate is now valid for localhost and the domain "hello.world.example"
+    let certs =
+        pemfile::certs(&mut cert.serialize_pem().unwrap().as_bytes()).expect("couldn't load certs");
+    let key = pemfile::pkcs8_private_keys(&mut cert.serialize_private_key_pem().as_bytes())
+        .expect("couldn't load rsa_private_key");
+    let addr = "0.0.0.0:1337".parse().unwrap();
+
+    let mut cert_file = tempfile::NamedTempFile::new().expect("Couldn't create cert file");
+    cert_file
+        .write_all(cert.serialize_pem().unwrap().as_bytes())
+        .expect("Couldn't write cert file");
+
+    let (server, received, shutdown_handle) = https_ingester(addr, certs, key[0].clone());
```

Comment:
This could be extracted to a reusable method and/or use https once, the rest of the test go through the http API.